engine/xml-config/src/main/resources/configuration.xsd

```diff
@@ -52,6 +52,9 @@
 				</choice>
 				<element name="storage-provider" type="ab:storageProviderType"
 					minOccurs="0" maxOccurs="1" />
+				<element name="descriptor-providers" type="ab:descriptorProvidersType" minOccurs="1" maxOccurs="1" />
```

Comment:
I guess ```minOccurs``` should be 0. Both to allow the deprecated variant, but also because in many cases (especially for conf.xml files on the monitor server) we would want the descriptor-provider not to be specified here.